Makefile

```diff
@@ -38,8 +37,18 @@ install-bin: bin/kind bin/kubectl-kuttl
 create-kind-cluster: $(KUBECONFIG)
 
 $(KUBECONFIG): install-bin
```

Comment:
Does this mean that if there's no kubeconfig envvar set, it will do all of these things?